src/Services/Masa.Auth.Service.Admin/Application/Subjects/Commands/AddStaffCommandValidator.cs

```diff
@@ -7,14 +7,6 @@ public class AddStaffCommandValidator : AbstractValidator<AddStaffCommand>
 {
     public AddStaffCommandValidator()
     {
-        RuleFor(command => command.Staff.JobNumber).Required().MaximumLength(20);
-        RuleFor(command => command.Staff.DisplayName).MaximumLength(50);
-        RuleFor(command => command.Staff.Name).ChineseLetter().MaximumLength(20);
-        RuleFor(command => command.Staff.PhoneNumber).Required().Phone();
-        RuleFor(command => command.Staff.Email).Email();
-        RuleFor(command => command.Staff.IdCard).IdCard();
-        RuleFor(command => command.Staff.CompanyName).ChineseLetter().MaximumLength(50);
-        RuleFor(command => command.Staff.Position).ChineseLetterNumber().MaximumLength(20);
-        RuleFor(command => command.Staff.Password).AuthPassword();
+        RuleFor(command => command.Staff).SetValidator(new AddStaffValidator());
```

Comment:
For example, a nickname is not required on the server side,for example:avatar、password,please refer to the source code to find out the difference,fix it.